src/app/components/doughnut-graph/doughnut-graph.component.ts

```diff
@@ -23,44 +24,25 @@ export class DoughnutGraphComponent implements OnInit {
       } else if (data.status == "none") {
         return ""
       } else {
-        var passedJobs = this.jobStatusCount(data.jobs,'success')
-        var failedJobs = this.jobStatusCount(data.jobs,'failed')
-        var canceledJobs = this.jobStatusCount(data.jobs,'canceled')
+        const passedJobs = this.jobStatusCount(data.jobs, 'success')
```

Comment:
We have enums for these strings . Lets pass enums.

Reply:
rewriting this component . will change this complete structure